src/main/java/no/nav/familie/integrasjoner/client/soap/PersonSoapClient.kt

```diff
@@ -2,19 +2,26 @@ package no.nav.familie.integrasjoner.client.soap
 
 import no.nav.familie.http.client.AbstractSoapClient
 import no.nav.familie.http.client.Pingable
-import no.nav.tjeneste.virksomhet.person.v3.binding.HentPersonhistorikkPersonIkkeFunnet
-import no.nav.tjeneste.virksomhet.person.v3.binding.HentPersonhistorikkSikkerhetsbegrensning
-import no.nav.tjeneste.virksomhet.person.v3.binding.PersonV3
+import no.nav.tjeneste.virksomhet.person.v3.binding.*
 import no.nav.tjeneste.virksomhet.person.v3.meldinger.HentPersonRequest
 import no.nav.tjeneste.virksomhet.person.v3.meldinger.HentPersonResponse
 import no.nav.tjeneste.virksomhet.person.v3.meldinger.HentPersonhistorikkRequest
 import no.nav.tjeneste.virksomhet.person.v3.meldinger.HentPersonhistorikkResponse
+import org.slf4j.LoggerFactory
+import org.springframework.retry.annotation.Backoff
+import org.springframework.retry.annotation.Recover
+import org.springframework.retry.annotation.Retryable
 import org.springframework.stereotype.Component
 import javax.xml.ws.soap.SOAPFaultException
 
+
 @Component
 class PersonSoapClient(private val port: PersonV3) : AbstractSoapClient("personV3"), Pingable {
 
+    private val LOGGER = LoggerFactory.getLogger(PersonSoapClient::class.java)
+
+    @Retryable(value = [SOAPFaultException::class], maxAttempts = 3, backoff = Backoff(delay = 2000))
```

Comment:
Er det greit at vi nå får 9 x retry på grunn av retry i fagsystemene også? Ooog det vil være "retry" på tasknivå.

Reply:
Tenker vi bør vurdere om det er hensiktsmessig å å ha retry i fagsystemene, der håndteres mye på tasknivå. Er også mindre risiko for nettverksbrudd og andre feilsituasjoner internt i clusteret.

Reply:
Helt enig! Vi kan godt ha policy om kun retry her i integrasjoner.